visualization/dashboard/src/MLIDashboard/Controls/DependencePlot.tsx

```diff
@@ -0,0 +1,364 @@
+import { ComboBox, IComboBox, IComboBoxOption } from "office-ui-fabric-react/lib/ComboBox";
+import * as Plotly from "plotly.js-dist";
+import React from "react";
+import { AccessibleChart, IPlotlyProperty, DefaultSelectionFunctions, PlotlyMode } from "mlchartlib";
+import { mergeStyleSets } from "@uifabric/styling";
+import { IDropdownOption } from "office-ui-fabric-react/lib/Dropdown";
+import { IconButton, Button, DefaultButton } from "office-ui-fabric-react/lib/Button";
+import { Transform } from "plotly.js-dist";
+import { IGenericChartProps, ISelectorConfig, ChartTypes } from "../NewExplanationDashboard";
+import { JointDataset, ColumnCategories } from "../JointDataset";
+import { IExplanationModelMetadata } from "../IExplanationContext";
+import { AxisConfigDialog } from "./AxisConfigurationDialog/AxisConfigDialog";
+import { localization } from "../../Localization/localization";
+import _ from "lodash";
+import { Cohort } from "../Cohort";
+import { Text } from "office-ui-fabric-react";
+import { FabricStyles } from "../FabricStyles";
+
+export interface INewDataTabProps {
+    chartProps: IGenericChartProps;
+    // selectionContext: SelectionContext;
+    theme?: string;
+    // messages?: HelpMessageDict;
+    jointDataset: JointDataset;
+    cohort: Cohort;
+    cohortIndex: number;
+    metadata: IExplanationModelMetadata;
+    onChange: (props: IGenericChartProps) => void;
+}
+
+export class DependencePlot extends React.PureComponent<INewDataTabProps> {
+    public static basePlotlyProperties: IPlotlyProperty = {
```

Comment:
Putting this comment here so that we can track it in the next PR assuming this is a #ToDo where we want to move the style to another file as well as remove the font and color to use themes.

Reply:
definitely, all the styles should be moved to a second file, and be themed.

Reply:
+1